src/components/FormEditor/FormEditor.js

```diff
@@ -6,27 +6,82 @@ import { Field } from 'redux-form';
 import { get, map, toPairs, fromPairs } from 'lodash';
 import { ValidatedField, MultiSelect } from '../Form';
 import * as Fields from '../../ui/components/Fields';
-import * as ArchitectFields from '../Form/Fields';
+import SelectOptionImage from '../Form/Fields/SelectOptionImage';
+import SelectOptionVariable from '../Form/Fields/SelectOptionVariable';
 import Guidance from '../Guidance';
 import Disable from '../Disable';
 import NodeType from './NodeType';
 import { getFieldId } from '../../utils/issues';
 import { actionCreators as dialogsActions } from '../../ducks/modules/dialogs';
 
+
 const allowedTypes = ['text', 'number', 'boolean', 'ordinal', 'categorical'];
 
+const inputDefinitions = {
+  NumberInput: {
+    label: 'Number Input',
+    value: 'NumberInput',
+    description: 'This input is optomized for collecting numerical data, and will show a number pad if available.',
+    image: 'TextInput',
+  },
+  Checkbox: {
+    label: 'Checkbox',
+    value: 'Checkbox',
+    description: 'This is a simple checkbox component that can be clicked or tapped to toggle a value between true or false.',
+    image: 'Checkbox',
+  },
+  CheckboxGroup: {
+    label: 'Checkbox Group',
+    value: 'CheckboxGroup',
+    description: 'This component provides a group of checkboxes so that multiple values can be toggled on or off.',
+    image: 'CheckboxGroup',
+  },
+  Toggle: {
+    label: 'Toggle',
+    value: 'Toggle',
+    description: 'This component renders a switch, which can be tapped or clicked to indicate "on" or "off".',
+    image: 'Toggle',
+  },
+  RadioGroup: {
+    label: 'Radio Group',
+    value: 'RadioGroup',
+    description: 'This will render a group of options and allow the user to choose one. Useful for likert-type scales or other ordinal variables.',
+    image: 'RadioGroup',
+  },
+  ToggleButton: {
+    label: 'Toggle Button',
+    value: 'ToggleButton',
+    description: 'This component provides a colorful button that can be toggled "on" or "off". It is useful for categorical variables where multiple options can be selected.',
+    image: 'ToggleButton',
+  },
+  ToggleButtonGroup: {
+    label: 'Toggle Button Group',
+    value: 'ToggleButtonGroup',
+    description: 'This component provides a colorful button that can be toggled "on" or "off". It is useful for categorical variables where multiple options can be selected.',
+    image: 'ToggleButtonGroup',
+  },
+  TextInput: {
+    label: 'Text Input',
+    value: 'TextInput',
```

Comment:
Network Canvas is looking for a 'Text' value (not 'TextInput'); should we change that here or in NC?